### PR TITLE
[refactor] Refactor types to a single folders

### PR DESCRIPTION
**Pull request by YuanmingLeee**
Close #

#### What has been done to verify that this works as intended?
N.A.

#### Why is this the best possible solution? Were any other approaches considered?
Consolidate all types in a single folder, like https://github.com/cortexlabs/cortex/tree/master/pkg/types

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Use `from modelci.types import xxx`

#### Does this change require updates to documentation?
Yes, updated.

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

**Comment by YuanmingLeee** [comment]
all type related files go to `modelci.types`

File: README.md [hunk 1]
```diff
@@ -112,20 +113,19 @@ MLModelCI provides a profile to automate the processing.
...
 ```python
-
 from modelci.hub.client.torch_client import CVTorchClient
 from modelci.hub.profiler import Profiler
 
 test_data_item = ...
 batch_num = ...
 batch_size = ...
-mode_info = ...
+model_info = ...
```
Comment (huangyz0918) [inline README.md:122]: 👍

File: modelci/types/bo/dynamic_profile_result_bo.py [hunk 2]
```diff
@@ -160,23 +160,29 @@ def __init__(self,
...
-                                        initialization_latency=self.latency.init_latency.tolist(),
-                                        preprocess_latency=self.latency.preprocess_latency.tolist(),
-                                        inference_latency=self.latency.inference_latency.tolist(),
-                                        postprocess_latency=self.latency.postprocess_latency.tolist(),
-                                        batch_formation_throughput=self.throughput.batch_formation_throughput.tolist(),
-                                        preprocess_throughput=self.throughput.preprocess_throughput.tolist(),
-                                        inference_throughput=self.throughput.inference_throughput.tolist(),
-                                        postprocess_throughput=self.throughput.postprocess_throughput.tolist(),
-                                        create_time=self.create_time)
+        dpr_po = DynamicProfileResultDO(
```
Comment (huangyz0918) [inline modelci/types/bo/dynamic_profile_result_bo.py:163]: Resolve #81 first so we can update the profiler's features.

**Comment by huangyz0918** [comment]
Please fix the code style issues before merging.

File: modelci/metrics/cadvisor/CLIENT.md [hunk 3]
```diff
@@ -26,74 +26,111 @@ You can use `-h` or `--help` to get more information.
...
 Init the cAdvisor client, you can set the API informations here.
 
+With default settings:
 ```python
-cAdvisor = CAdvisor() # init with default settings
-cAdvisor = CAdvisor(base_link='http://localhost:8080/api/', ersion='v1.2', query_object='docker') 
-# init with specific API dettings
+from modelci.metrics.cadvisor.cadvisor import CAdvisor
+
+c_advisor = CAdvisor()
```
Comment (HuaizhengZhang) [inline modelci/metrics/cadvisor/CLIENT.md:33]: so weired.

container_monitor is better

File: modelci/types/bo/dynamic_profile_result_bo.py (hunk 2 above)
Comment (HuaizhengZhang) [reply modelci/types/bo/dynamic_profile_result_bo.py:163]: agree

Check issue first

Comment (YuanmingLeee) [reply modelci/types/bo/dynamic_profile_result_bo.py:163]: will do after this merge

**Pull request merged**